### ISSUE TITLE
Navigation tags wrong

### ISSUE THREAD

**Issue by Pfeifakopf** [statamic/cms#1560]
After updating from beta20 to beta22 is the output of all tags inside the nav:collection wrong.
```
{{ nav:breadcrumbs }}
{{ nav:collection:products :from="url" }}
{{ title }}
{{ /nav:collection:products  }}
{{ /nav:breadcrumbs }}`
```

**Comment by jackmcdade** [comment]
Can you explain this a bit please? Why are you nesting a nav tag inside a breadcrumb tag?

**Comment by Pfeifakopf** [comment]
I use it to navigate directly through the navigation by hovering the breadcrumbs. It's totally fast and intuitiv.  Like this:
![image](https://user-images.githubusercontent.com/3351449/77827659-370ce580-7117-11ea-9288-bcb85650a9dd.png)

After the update the count of the entries for each navigation point is right, but the tags like title, url, etc. giving the content from the breadcrumb point, i think. Like this:
Before update:
```
Breadcrumb
 - Child 1 
 - Child 2
 - Child 3
```
After update:
```
Breadcrumb
 - Breadcrumb
 - Breadcrumb
 - Breadcrumb
```
Hope it's clearer now. Otherwise i could show it to you on discord.

**Comment by jackmcdade** [comment]
Fixed in 34072840f7894f454f7b6ec264a07b1fb440eb08.
